docs/index.rst

```diff
@@ -12,6 +12,7 @@ Welcome to xaitk-saliency's documentation!
 
    installation
    releasing
+   design
    miscelaneous
```

Comment:
Minor spelling nitpick: `miscelaneous` -> `miscellaneous`.

Might have to update the spelling in the corresponding rst file as well (i.e. miscelaneous.rst).

Reply:
Must have missed the spelling squiggles on this. Will correct.